decidim-core/app/models/decidim/user_group.rb

```diff
@@ -202,13 +202,13 @@ def has_document_number?
       document_number.present?
     end
 
-    # Overwites method in `Decidim::Validatable`, as user groups do not have a
+    # Overrides method in `Decidim::Validatable`, as user groups do not have a
     # password.
     def password_required?
       false
     end
 
-    # Overwites method in `Decidim::Validatable`, as user groups do not have a
+    # Overrides method in `Decidim::Validatable`, as user groups do not have a
```

Comment:
It's possible that `Overwrites` is intended, but I think that this is the correct fix.